types/puppeteer/index.d.ts

```diff
@@ -1447,6 +1443,16 @@ export interface Page extends EventEmitter, FrameBase {
    */
   waitForNavigation(options?: NavigationOptions): Promise<Response>;
 
+  waitForRequest(
```

Comment:
This change seems like it was committed by accident. It just moves the functions around.

Reply:
Thanks for the review! I moved these functions because they are [part of the Page class](https://github.com/GoogleChrome/puppeteer/blob/3762de7b91bb85d2f1979d49152af79a92a06bbd/lib/Page.js#L635-L665) and don't exist on [Frames](https://github.com/GoogleChrome/puppeteer/blob/3762de7b91bb85d2f1979d49152af79a92a06bbd/lib/FrameManager.js#L253).  Since they were introduced in 1.6, I thought it was kind of consistent to keep this fix in a single PR.

Reply:
@slyrz ohh right, I totally missed that. Thanks for clearing it up :)